palm/environment.py

```diff
@@ -49,10 +49,13 @@ def run_in_shell(self, cmd: str, env_vars: Optional[dict] = {}):
         click.secho(msg, fg="green" if success else "red")
 
     def run_on_host(
-        self, cmd: str, bubble_error: Optional[bool] = False
+        self,
+        cmd: str,
+        bubble_error: Optional[bool] = False,
```

Comment:
help me understand, why is bubble error set to false?

Reply:
That is the default value, this attr should probably be renamed `check` so that it is clearly alligned with the underlying attr in `subprocess.run` - the default value of check is False, so our implementation aligns with the implementation of subprocess.run

Reply:
yeah `bubble_error` felt like it was more intuitive.. but maybe it's not. actually it's probably not. meh.